src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/WalkerTopLevelCommandItem.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data;
+using System.IO;
+using Microsoft.CmdPal.Ext.WindowWalker.Helpers;
+using Microsoft.CmdPal.Ext.WindowWalker.Pages;
+using Microsoft.CmdPal.Ext.WindowWalker.Properties;
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.WindowWalker;
+
+public partial class WalkerTopLevelCommandItem : CommandItem
+{
+    public WalkerTopLevelCommandItem(SettingsManager settingsManager)
```

Comment:
For now I'm going to just return a new commanditem each time the command is invoked. The other mechanism doesn't seem to work well today and I'm not sure if it will get fixed in TRA.

So, done 🫡.